pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/KitRequestShipping.java

```diff
@@ -90,7 +90,7 @@
 @SuperBuilder(setterPrefix = "with", toBuilder = true)
 public class KitRequestShipping extends KitRequest implements HasDdpInstanceId {
 
-    public static final String SQL_SELECT_KIT_REQUEST =
+    public static final String SQL_SELECT_KIT_REQUEST_FOR_ES =
```

Comment:
this SQL is used only in kit relates ES export . minimizes test impact and exports de-activates records even though they were re-activated so that history is maintained in elastic.